cmake/leveldb.cmake

```diff
@@ -0,0 +1,15 @@
+include(FetchContent)
+FetchContent_Declare(
+    leveldb
+    GIT_REPOSITORY 	https://github.com/helix-re/leveldb.git
+	GIT_TAG 		helix-master
```

Comment:
Are we making changes to level DB?. Why do we have helix-master here?

Reply:
leveldb has many subprojects for tests, utilities and benchmarks. 
In helix-master I kept only required. 
Only `CMakeLists.txt` is changed.